php/libraries/Candidate.class.inc

```diff
@@ -139,6 +139,15 @@ class Candidate
                     $params[$row['Description']] = $row['Value'];
```

Comment:
Nitpick: instead of using the `array_map` call below, wouldn't it be simpler to just sanitize the values of `$params` right when they are created:

`$param[$row['Description']] = htmlentities($row['Value'], ENT_QUOTES, 'UTF-8');`

Reply:
Sure that seems simpler. I'll update my fix

Reply:
Done!